fastFM/transform.py

```diff
@@ -1,7 +1,7 @@
 # Author: Immanuel Bayer
 # License: BSD 3 clause
 
-import scipy.sparse as sp
+import scipy.sparse as sparse
```

Comment:
Please stick with ``import scipy.sparse as sp``. I think this is quite common and used in all the other modules.